include/dxc/DxilContainer/RDAT_LibraryTypes.inl

```diff
@@ -9,15 +9,19 @@
 //                                                                           //
 ///////////////////////////////////////////////////////////////////////////////
 
+// clang-format off
```

Comment:
If we were going to clang-format off this file, we should have done it in `main` first to reduce the diff here.

Reply:
https://github.com/microsoft/DirectXShaderCompiler/pull/5986

Reply:
Pull request to turn off clang formatting is completed in main and this PR was updated with a merge from main that contains this commit.